admin/partials/acc_user_importer-admin-settings.php

```diff
@@ -103,8 +103,19 @@ function accUM_settings_init () {
 			array(
 				'type' => 'text',
 				'name' => 'accUM_memberURI',
-				'html_tags' => 'required',
-				'default' => 'Asi.Scheduler_DEV/api/IQA?QueryName=$/ACC/Queries/REST/ACC_VA'
```

Comment:
I am not sure why you are removing the above 2 lines.  Can you please explain?

Reply:
I don't think it is required, you need to get the URL from Jeffrey anyways, so it should be straightforward what goes in there. I don't like defaults that don't work. If it is not clear, we could add a description field and explain more what the URL should look like.